<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScpiRfm2gDataCollectToolClass</class>
 <widget class="QMainWindow" name="ScpiRfm2gDataCollectToolClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ScpiRfm2gDataCollectTool</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget{
background:#eeeeee;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">#widget_left,#widget_top,#widget_bottom {
border:1px solid gray;
border-radius:2px;
}
QWidget{
font-family:&quot;微软雅黑&quot;;
font-size:12px;
}
QCheckBox{
spacing: 5px;
font-size:20px;
}
QCheckBox::indicator {
      width: 44px;
      height: 32px;
}
QCheckBox::indicator:unchecked{
image: url(:/ScpiRfm2gDataCollectTool/Resources/unchecked.png);
}
QCheckBox::indicator:checked{
image: url(:/ScpiRfm2gDataCollectTool/Resources/checked.png);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
    <property name="spacing">
     <number>16</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>25</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QWidget" name="widget_left" native="true">
      <property name="styleSheet">
       <string notr="true">#tableWidget{
border:1px solid gray;
}
QTableView{
selection-background-color: gray;
selection-color:#fff;
}
QTableView QTableCornerButton::section{
background: red;
border: 2px outset red;
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>6</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>全选</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_status">
           <property name="minimumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background:gray;
border-radius:12px;</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">font-size:18px;
padding-right:6px;</string>
           </property>
           <property name="text">
            <string>传输通道选择</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget"/>
       </item>
       <item>
        <widget class="QTextEdit" name="textEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>120</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border:1px solid gray;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2">
      <property name="spacing">
       <number>17</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_top" native="true">
        <property name="styleSheet">
         <string notr="true"> QLineEdit {
      border: 1px solid #8f8f91;
      border-radius: 2px;
      padding: 0 8px;
      background: #fff;
      selection-background-color: darkgray;
  }</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>9</number>
         </property>
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>发送周期：</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lineEdit_timeInterval">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>32</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>毫秒</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>绑定网卡：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_bindIp">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>32</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_bottom" native="true">
        <property name="styleSheet">
         <string notr="true">
QPushButton{
      border: 1px solid #8f8f91;
      border-radius: 4px;
      background-color: qlineargradient(x1: 0, y1: 0, x2:0, y2: 1,stop: 0 #f6f7fa, stop: 1 #dadbde);
      min-width: 80px;
  
  }

  QPushButton:pressed {
      background-color: qlineargradient(x1: 0, y1: 0, x2:0, y2: 1,stop: 0 #dadbde, stop: 1 #f6f7fa);
  }

  QPushButton:flat {
      border: none; /* no border for a flat push button */
  }

  QPushButton:default {
      border-color: navy; /* make the default button prominent */
  }</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>12</number>
         </property>
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="rightMargin">
          <number>12</number>
         </property>
         <item>
          <spacer name="verticalSpacer_3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>33</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="button_connect">
           <property name="minimumSize">
            <size>
             <width>82</width>
             <height>32</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>配置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_start">
           <property name="minimumSize">
            <size>
             <width>82</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>开始</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_stop">
           <property name="minimumSize">
            <size>
             <width>82</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>停止</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_exit">
           <property name="minimumSize">
            <size>
             <width>82</width>
             <height>32</height>
            </size>
           </property>
           <property name="text">
            <string>退出</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>34</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ScpiRfm2gDataCollectTool.qrc"/>
 </resources>
 <connections/>
</ui>
